docs/docs/components.mdx

```diff
@@ -615,6 +615,13 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
   end-to-end before trying to optimize this component with other weights/architectures.
   :::
 
+  :::note
+  While we make an effort to support as many HuggingFace models as possible, please note that some architectures will
+  not work due to how Rasa's internal tokenization interacts with the HuggingFace tokenizers. If you encounter an issue
+  with a specific model other than the ones listed in the table above, please let us know on
```

Comment:
Maybe we can sketch to them how to check whether a tokenizer will work?

Reply:
Could even link to the unit test. I'd still hedge this, meaning I wouldn't promise that if they get good results on one test it will work perfectly

Reply:
SImilarly, I think it would help if we listed known problematic models:
CRTL, BERTweet, PhoBERT?